cmd/sys-mgmt-executor/main.go

```diff
@@ -17,45 +17,33 @@ import (
 	"errors"
 	"flag"
 	"fmt"
+	"github.com/edgexfoundry/go-mod-core-contracts/clients/logger"
```

Comment:
please run `gofmt` on this file so that the github imports are separate from the std library imports and are alphabetized. `gofmt` will do this for you